src/InstructionWalker.h

```diff
@@ -245,6 +245,8 @@ namespace vc4c
 			return func(get());
 		}
 
+		void replaceLocal(const Local *oldLocal, const Local *newLocal, bool forward=true, bool stopFlag=true);
```

Comment:
Can you please add some documentation to this functions, especially replace, since the name is not very specific. Also, please explain what the `stopFlag` is for.
Another thing that should be in the documentation is, that the values are not replaced in the instruction itself, but only in the succeeding/preceding within the same block